UNIXFS.md

```diff
@@ -227,7 +431,69 @@ Fractional values are effectively a random number in the range 1 ~ 999,999,999.
 2^28 nanoseconds ( 268,435,456 ) in most cases. Therefore, the fractional part is represented as a 4-byte
 `fixed32`, [as per Google's recommendation](https://developers.google.com/protocol-buffers/docs/proto#scalar).
 
-[multihash]: https://tools.ietf.org/html/draft-multiformats-multihash-00
-[CID]: https://docs.ipfs.io/guides/concepts/cid/
+## References
+
+[multihash]: https://tools.ietf.org/html/draft-multiformats-multihash-05
+[CID]: https://github.com/multiformats/cid/
 [Bitswap]: https://github.com/ipfs/specs/blob/master/BITSWAP.md
-[MFS]: https://docs.ipfs.io/guides/concepts/mfs/
+
+# Notes for Implementers
+
+This section and included subsections are not authoritative.
+
+## Implementations
+
+- JavaScript
+  - Data Formats - [unixfs](https://github.com/ipfs/js-ipfs-unixfs)
+  - Importer - [unixfs-importer](https://github.com/ipfs/js-ipfs-unixfs-importer)
+  - Exporter - [unixfs-exporter](https://github.com/ipfs/js-ipfs-unixfs-exporter)
```

Comment:
Should we also add https://github.com/ipld/js-unixfs?

Reply:
https://github.com/ipld/js-unixfs ✅ - yes we use this for encoding UnixFS DAGs now
https://github.com/web3-storage/fast-unixfs-exporter ❌ - temporary fork now deprecated